cmake/External/xccl.cmake

```diff
@@ -0,0 +1,13 @@
+if(NOT __XCCL_INCLUDED)
+  set(__XCCL_INCLUDED TRUE)
+
+  if(USE_XCCL)
+    # XCCL_ROOT, XCCL_LIBRARY_DIR, XCCL_INCLUDE_DIR are handled by FindXCCL.cmake.
+    find_package(XCCL REQUIRED)
+    if(XCCL_FOUND)
+      add_library(__caffe2_xccl INTERFACE)
```

Comment:
PyTorch is going to remove caffe2, `__caffe2_xxl` seems not a good name. I prefer `torch::xccl`

Reply:
done, use torch::xccl, align with torch::xpurt.